src/rsa_primitive_verify_tests.txt

```diff
@@ -0,0 +1,5 @@
+# Big-endian public key parts for signature_rsa_example_public_key.der,
+# encoded in hex.
+
+n = CEA80475324C1DC8347827818DA58BAC069D3419C614A6EA1AC6A3B510DCD72CC516954905E9FEF908D45E13006ADF27D467A7D83C111D1A5DF15EF293771AEFB920032A5BB989F8E4F5E1B05093D3F130F984C07A772A3683F4DC6FB28A96815B32123CCDD13954F19D5B8B24A103E771A34C328755C65ED64E1924FFD04D30B2142CC262F6E0048FEF6DBC652F21479EA1C4B1D66D28F4D46EF7185E390CBFA2E02380582F3188BB94EBBF05D31487A09AFF01FCBB4CD4BFD1F0A833B38C11813C84360BB53C7D4481031C40BAD8713BB6B835CB08098ED15BA31EE4BA728A8C8E10F7294E1B4163B7AEE57277BFD881A6F9D43E02C6925AA3A043FB7FB78D
```

Comment:
So I'm back to working on this PR now, and I think I have the other comments covered, but the testing here is still hard for me to wrap my head around. Can you lay out the test matrix in a table or list a bit more, please? In particular, how do I create keys that have the maximum or minimum value for `n` (which is the public exponent, right?)? Isn't the size of `n` now covered by the `min_bits` property of `RSAParameters`?